tests/lib/rules/prop-types.js

```diff
@@ -1588,15 +1588,6 @@ ruleTester.run('prop-types', rule, {
         }
       `,
     },
-    {
-      // Async generator functions can't be components.
-      code: `
-        var Hello = async function* (props) {
-          yield null;
-          return <div>Hello {props.name}</div>;
-        }
-      `,
-    },
```

Comment:
Duplicate of:

https://github.com/jsx-eslint/eslint-plugin-react/blob/8e1a94b67d081fdc132e9a7e175db3fbf2e02956/tests/lib/rules/prop-types.js#L1573-L1581